tools/deployment/make_osx_package.sh

```diff
@@ -50,42 +39,22 @@ NEWSYSLOG_SRC="$SCRIPT_DIR/$LD_IDENTIFIER.conf"
 NEWSYSLOG_DST="/private/var/osquery/$LD_IDENTIFIER.conf"
 PACKS_SRC="$SOURCE_DIR/packs"
 PACKS_DST="/private/var/osquery/packs/"
-LENSES_LICENSE="${OSQUERY_DEPS}/Cellar/augeas/*/COPYING"
-LENSES_SRC="${OSQUERY_DEPS}/share/augeas/lenses/dist"
+LENSES_LICENSE="libs/fb/augeas/augeas/1.9.0/COPYING"
```

Comment:
If we continue using this script it would be nice to pull these Augeas paths from the CMake discovery.

Reply:
II think a better approach would be to `make DESTDIR="${INSTALLPREFIX}" install` instead of manually copying the files. This would future-proof this script against any modification on the build system

Reply:
I think so to, my plan is to fix these scripts then follow up to change the API (parameters) to use `make install` approach.

Reply:
I'll track that next change here: #5842